tests/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater.Test/ConfigUpdaterTest.cs

```diff
@@ -0,0 +1,186 @@
+                                            </behavior>
+                                          </serviceBehaviors>
+                                        </behaviors>-->
+                                        <services>
+                                            <service name=""SampleServices""  behaviorConfiguration=""SampleBehavior"">
+                                            <endpoint address=""/SampleService"" binding=""netTcpBinding"" contract=""SampleService.IService""/>
+                                            <endpoint address=""/sample/address/SampleService2"" binding=""basicHttpsBinding"" contract=""SampleService.IService""/>
+                                            <!--The mex endpoint is removed because it's not support in CoreWCF. Instead, the metadata service is enabled in the source code.-->
+                                            <!--The host element is not supported in configuration in CoreWCF. The port that endpoints listen on is instead configured in the source code.-->
+                                            <!--<host>
+                                                <baseAddresses>
+                                                <add baseAddress = ""http://localhost:80/""/>
+                                                <add baseAddress = ""https://localhost:443/sample/address""/>
+                                                <add baseAddress= ""net.tcp://localhost:808/""/>
+                                                </baseAddresses>
+                                            </host>-->
+                                            </service>
+                                        </services>
+                                        </system.serviceModel>
+                                        </configuration>";
+
+        private readonly NullLogger<ConfigUpdater> _logger = NullLogger<ConfigUpdater>.Instance;
+
+        [Fact]
+        public void GetUriTest()
+        {
+            Dictionary<string, Uri> expected = new Dictionary<string, Uri>();
+            expected.Add(Uri.UriSchemeHttp, new Uri("http://localhost:80/"));
+            expected.Add(Uri.UriSchemeHttps, new Uri("https://localhost:443/sample/address"));
+            expected.Add(Uri.UriSchemeNetTcp, new Uri("net.tcp://localhost:808/"));
+
+            var result = new ConfigUpdater(XDocument.Parse(Input), _logger).GetUri();
+            Assert.Equal(expected, result);
+        }
+
+        [Fact]
+        public void GetBindingsTest()
+        {
+            HashSet<string> expected = new HashSet<string>();
+            expected.Add("netTcpBinding");
+            expected.Add("basicHttpsBinding");
+            expected.Add("mexHttpBinding");
+            var result = new ConfigUpdater(XDocument.Parse(Input), _logger).GetBindings();
+            Assert.Equal(expected, result);
+        }
+
+        [Theory]
+        [InlineData("", false)]
+        [InlineData(ServiceDebug, true)]
```

Comment:
Use string.Empty instead of ""

Reply:
The attribute argument here has to be a constant expression and it's not letting me use string.Empty

Reply:
Yup that's correct - can't use `string.Empty` here.